docs/src/app/(public)/docs/layout.tsx

```diff
@@ -5,8 +5,10 @@ import { DocsFooter } from '../../../components/footer';
 import { getNavigationMap } from '../../../utils/reader';
 
 export const metadata = {
-  title: 'Keystatic - Docs',
-  description: 'Documentation for Keystatic.',
+  title: {
+    template: '%s - Docs | Keystatic',
+    default: 'Docs',
```

Comment:
Not `Docs | Keystatic`?

Reply:
Doesn't seem to be needed 🤔 If I do that then it renders a title with `Blog | Keystatic | Keystatic` if the `/blog/[...slug]/page.tsx` has no meta data `export const metadata = {};`

So I guess it takes the default, and then the template from the parent layout which is just `template: '%s - | Keystatic',`

Reply:
>  it takes the default, and then the template from the parent layout

Ahh yes, you're absolutely right.

Re-reading the [Next docs on metadata](https://nextjs.org/docs/app/api-reference/functions/generate-metadata#template):

> `title.template` applies to child route segments and not the segment it's defined in.